ocs_ci/ocs/constants.py

```diff
@@ -987,6 +987,8 @@
 RHV_PLATFORM = "rhv"
 ROSA_PLATFORM = "rosa"
 FUSIONAAS_PLATFORM = "fusion_aas"
+HCI_BAREMETAL = "hci_baremetal"
+HCI_VSPHERE = "hci_vsphere"
```

Comment:
just not sure if it is correct.
From OCP perspective the cluster is a BM cluster. And it is not an HCI, I think HCI deployed another way.
<img width="558" alt="image" src="https://github.com/red-hat-storage/ocs-ci/assets/61454420/09da31cf-3f59-4feb-a8d0-b99476768136">

Reply:
The reason for introducing this type is - when node-specific tests will attempt for execution , we can differentiate the platform API to interact with. Like if we are using vpshere and installing like a baremetal , even though to interact with hardware API we need a vsphere-specific details. Similarly we need credential files for vpshere or baremetal depending on use of the actual base platform

Reply:
As per my understanding, we are supporting only baremetal platform, at least for phase-1 GA . However, QE are setting cluster using Vsphere or IBMcloud as baremetal cluster . so at the top level it is baremetal platform however H/w interaction is depend on actual platform only